test_database.py

```
@@ -25,10 +25,9 @@ def test_database_init():
     try:
         database.init_database()
         logger.info("Database initialized successfully")
-        return True
     except Exception as e:
         logger.error(f"Database initialization failed: {str(e)}", exc_info=True)
-        return False
+        assert False, f"Database initialization failed: {str(e)}"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)